lib/etc/openturns.conf.in

```diff
@@ -796,12 +796,22 @@
   <FunctionalChaosSobolIndices-MaximumNumberOfOutput value_int="1000" />
   <FunctionalChaosSobolIndices-PrintColumnWidth value_int="15" />
 
+  <!-- OT::FunctionalChaosValidation parameters -->
+  <FunctionalChaosValidation-DefaultKFoldParameter value_int="10" />
+  <FunctionalChaosValidation-NoModelSelection      value_bool="true" />
+
   <!-- OT::LeastSquaresExpansion parameters -->
   <LeastSquaresExpansion-DecompositionMethod       value_str="QR" />
 
   <!-- OT::LinearModelAlgorithm parameters -->
-  <LinearModelAnalysis-Identifiers                 value_int="3"      />
-  <LinearModelAlgorithm-DecompositionMethod        value_str="QR"      />
+  <LinearModelAlgorithm-DecompositionMethod        value_str="QR" />
+
+  <!-- OT::LinearModelAnalysis parameters -->
+  <LinearModelAnalysis-Identifiers                 value_int="3" />
```

Comment:
IdentifiersNumber would be more aligned with other key names in ResourceMap

Reply:
Fixed at 59ac13017